neurom/fst/tests/test_neuritefunc.py

```diff
@@ -186,3 +186,14 @@ def test_volume_density_per_neurite():
     ref_density = [0.43756606998299519, 0.52464681266899216,
                    0.24068543213643726, 0.26289304906104355]
     nt.ok_(np.allclose(vol_density, ref_density))
+
+
+def test_terminal_length_per_neurite():
+    nrn = nm.load_neuron(os.path.join(SWC_PATH, 'simple.swc'))
+    terminal_distances = np.array(_nf.terminal_path_lengths_per_neurite(nrn))
+    np.testing.assert_allclose(terminal_distances,
+                               np.array([4 + 5., 4 + 6., 4. + 6., 4. + 5]))
+    terminal_distances = np.array(_nf.terminal_path_lengths_per_neurite(
+        nrn, neurite_type=nm.AXON))
```

Comment:
From what I see in the swc, the result would be the same with nm.DENDRITE right?
can we have a difference somewhere between the 2 neurites ?